server/server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+let app = express();
```

Comment:
I would recommend you put non require statements after _all_ your require statements

```suggestion
 const app = express();
```